src/api/stats/bsc/wault/getWaultLpApys.js

```diff
@@ -0,0 +1,65 @@
+const BigNumber = require('bignumber.js');
+const { bscWeb3: web3 } = require('../../../../utils/web3');
+
+const MasterChef = require('../../../../abis/WaltMaster.json');
+const fetchPrice = require('../../../../utils/fetchPrice');
+const pools = require('../../../../data/waultLpPools.json');
+const { compound } = require('../../../../utils/compound');
+const { getTotalLpStakedInUsd } = require('../../../../utils/getTotalStakedInUsd');
+const { BSC_CHAIN_ID } = require('../../../../constants');
+const getBlockNumber = require('../../../../utils/getBlockNumber');
+
+const getWaultLpApys = async () => {
```

Comment:
as there are many LPs make sense to copy multichain version (cake or ape), it should be 1:1.

or there is a degen/getMasterChefApy
https://github.com/beefyfinance/beefy-api/blob/212fb23eaf5fd5b9a807acc3991f8255258fb71e/src/api/stats/bsc/jetfuel/getJetswapApys.js#L6

hope to make it multichain one day